pyincore/dfr3service.py

```diff
@@ -11,6 +11,7 @@
 import requests
```

Comment:
Unused import and can you fix the pep8 issue with logger being initialized between the library imports? It gives a pep8 error since they are broken up.